common-rendering/src/components/accordion.stories.tsx

```diff
@@ -0,0 +1,85 @@
+// ----- Imports ----- //
+
+import Accordion from "./accordion";
+import { css, type SerializedStyles } from "@emotion/react";
+import { breakpoints, space } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+import { body } from '@guardian/src-foundations/typography';
+import type { FC } from "react";
+
+// ----- Stories ----- //
+
+const textStyle: SerializedStyles = css`
+	${body.medium()};
+	line-height: 150%;
```

Comment:
Two questions on this:

- Is this the same as `body.medium({ lineHeight: 'loose' })`?
- Is "loose" (i.e. 1.5, 150%) the default for `body` already?

Reply:
I thought loose was 135% but I could be wrong!

Reply:
Nope, you're right - that is loose 👍🏻